docs/plugins/directives.mjs

```diff
@@ -0,0 +1,102 @@
+import { u } from "unist-builder";
```

Comment:
this is copied from the myst docs, right? can you put a comment at the top that notes where it was copied from? Is there no way to just re-use the mystmd plugin? We have to copy/paste?

Reply:
Is there an issue tracking this that we can track? Then you could just add a comment like "Copied from XXX because we can't yet load external plugins, see <link to issue> for more detail"

Reply:
Certainly; https://github.com/jupyter-book/mystmd/issues/1101. I'll modify the source!